enhancements/installer/ipi-aws-china.md

```diff
@@ -0,0 +1,132 @@
+---
+title: ipi-install-aws-china
+authors:
+  - "@wanghaoran1988"
+reviewers:
+  - TBD
+approvers:
+  - TBD
+creation-date: 2020-02-14
+last-updated: 2020-02-14
+status: implementable
+---
+
+# IPI install on AWS China
+
+## Release Sign off Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+We have customers want to install OpenShift on AWS China, but currently
+OpenShift installer doesn't show up the AWS China regions `cn-north-1` and `cn-northwest-1`.
+These two regions are isolated from other global regions and RHCOS AMI pushed to global cannot
+be used in AWS China, [ARNs and api endpoint](https://docs.amazonaws.cn/en_us/aws/latest/userguide/endpoints-arns.html) are different.
+
+Similar to other global regions, we should support these two regions for customer who want deploy
+OpenShift on AWS China.
+
+## Motivation
+
+### Goals
+
+* OpenShift installer support IPI install on AWS China Regions.
+* CI job executing testings on AWS China regions.
+
+### Non-Goals
+
+* It's not a goal to detail how to request and setup a AWS account in AWS China.
+* It's not a goal to detail how to do UPI install.
+* It's not a goal to support all AWS service endpoints that are not part of the SDK for AWS China Regions.
+
+## Proposal
```

Comment:
Do we need to worry about the Great Firewall? Quay uses S3 as it’s backing store and China blocks it from time to time. It’s been a while since I’ve had to pay attention to that, so maybe it’s not an issue anymore.

Reply:
Quay.io is slow, but it's still workable at the moment.

Reply:
Okay, that doubles my concern then. Slow download speeds are going to have a large impact on cluster install times and their ability to scale quickly. And there still seems to be the possibility that the firewall will block the traffic outright.

Reply:
Quay has a georeplication project underway which *might* help, but we cannot depend on that as a prerequisite for this China bringup, especially since so far in China testing we haven't had any failures.  Also, we're not do not expect a large volume of cluster installs or scaling activity in China.

Reply:
Wait quay really uses S3 and not a proper CDN that would have regional distribution?  Hmm, AFAICS I'm not getting any redirects for the blobs indeed.